src/platform_linux/platform.h

```diff
@@ -278,28 +293,67 @@ typedef uint32 (*hash_fn)(const void *input, size_t length, unsigned int seed);
  * DO NOT USE these macros to assign to a void*.  The debug asserts will cause
  * a compile error when debug is on.  Assigning to a void* should be done by
  * calling aligned_alloc manually (or create a separate macro)
+ *
+ * Parameters:
+ *	id  - Platform heap-ID to allocate memory from.
```

Comment:
While doing this cleanup, I **really** would have liked to rename this `id` parameter to `hid` as that's typically what this variable **is** named when passed-around through other interfaces.

I think the change **will be** more illustrative and consistent with use of `hid` elsewhere in the code.

The change is simple to do ... but I wanted to get your, @ajhconway , approval before doing this roto-tilling.

Reply:
Done. Preparing for check-in ...